### ISSUE TITLE
Possible to add a confirm email box when registering for accounts

### ISSUE THREAD

**Issue by ericvanboven** [drlippman/IMathAS#111]
Is it possible to add a confirm email box when registering for accounts. We have it set so that a verification email goes out and some students are typing the email wrong.

**Comment by drlippman** [comment]
You mean a "type it a second time" box?

You certainly could try adding one, though I wonder how useful it would be. I always just copy-paste those fields, so wonder if many people actually retype it.

I've always thought a "sign in with Google / Facebook" would be a nice option...

**Comment by ericvanboven** [comment]
Yes. I do the same too but a faculty was asking for the request. At that
point if a student types email wrong and copies it then least we did our
due diligence in making sure they typed it in right.
we went to cloud azure training and they were showing
Facebook/google/twitter as a login like that where it can store those in a
dif public cloud authentication section in addition to normal business
users. Pretty cool stuff though i have to imagine it goes against using the
same password for multi services strategy where if they phish facebook info
they got access to more than just facebook (though im sure students just
use the same passwords anywas)

On Jan 18, 2018 8:16 AM, "David Lippman" <notifications@github.com> wrote:

You mean a "type it a second time" box?

You certainly could try adding one, though I wonder how useful it would be.
I always just copy-paste those fields, so wonder if many people actually
retype it.

I've always thought a "sign in with Google / Facebook" would be a nice
option...

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/drlippman/IMathAS/issues/111#issuecomment-358677487>,
or mute the thread
